purchases_flutter_ui/lib/purchases_flutter_ui.dart

```diff
@@ -0,0 +1,20 @@
+
+import 'package:flutter/services.dart';
+
+class RevenueCatUI {
```

Comment:
I modified the generated code from the wizard to resemble more closely what we do in the existing plugin. The generated code changed things so everything wasn't all static methods and better supported different communication systems with the platform (other than methodchannel). But it felt more complex so I simplified it like this.